tests/framework/common.go

```diff
@@ -88,6 +88,14 @@ func OSMDescribe(name string, opts OSMDescribeInfo, body func()) bool {
 	return Describe(fmt.Sprintf("%s %s", opts, name), body)
 }
 
+const (
+	// AppPort is the port to be used for server and client applications in e2es
+	AppPort = 14001
+)
+
+// AppCmd is the command to be used for server and client applications in e2es
```

Comment:
I'd call this something like `HttpbinCmd` since it's unique to httpbin.

Also since clients don't use httpbin:
```suggestion
// AppCmd is the command to be used for httpbin applications in e2es
```